src/package/packageVersionCreate.ts

```diff
@@ -343,17 +291,20 @@ export class PackageVersionCreate {
     const packageVersTmpRoot = path.join(os.tmpdir(), `${uniqueHash}`);
     const packageVersMetadataFolder = path.join(packageVersTmpRoot, 'md-files');
     const unpackagedMetadataFolder = path.join(packageVersTmpRoot, 'unpackaged-md-files');
+    const seedMetadataFolder = path.join(packageVersTmpRoot, 'seed-md-files');
     const packageVersProfileFolder = path.join(packageVersMetadataFolder, 'profiles');
     const packageVersBlobDirectory = path.join(packageVersTmpRoot, 'package-version-info');
     const metadataZipFile = path.join(packageVersBlobDirectory, 'package.zip');
     const unpackagedMetadataZipFile = path.join(packageVersBlobDirectory, 'unpackaged-metadata-package.zip');
+    const seedMetadataZipFile = path.join(packageVersBlobDirectory, 'seed-metadata-package.zip');
```

Comment:
if your program is working, i would like to see the structure of the zip file with seed-metadata in it.  you may share this on gdrive or slack. for one, it will help me working the core side changes.  

when ready, i would like to use these changes locally to start my tests.  It's harder to construct the binary data to use tooing api directly.